.github/workflows/schedule.yaml

```diff
@@ -32,4 +32,4 @@ jobs:
       - uses: 'actions/checkout@v4'
         with:
           sparse-checkout: '.github'
-      - run: 'python3 .github/workflows/schedule.py'
+      - run: 'python .github/workflows/schedule.py'
```

Comment:
Check which python versions GH's action image includes and how they're invoked. If it includes python 2 for example, how do we make sure that `python` calls version 3 and not 2?

Reply:
From the base image's public [beta release issue](https://github.com/actions/runner-images/issues/5490) (we're using Ubuntu 22.04) for Python…
Default version: 3.10.x
Pre-cached versions: 3.7, 3.8, 3.9, 3.10

We would ensure that the calls are to version 3 by removing the 2 version from the image.